src/test/scala/io/iohk/ethereum/ledger/LedgerSpec.scala

```diff
@@ -197,7 +197,7 @@ class LedgerSpec extends FlatSpec with PropertyChecks with Matchers with MockFac
         val BlockResult(resultingWorldState, resultingGasUsed, resultingReceipts) = txsExecResult.right.get
 
         //Check valid world
-        val minerPaymentForTxs = UInt256(stx.tx.gasLimit * stx.tx.gasPrice)
+        val minerPaymentForTxs = UInt256(stx.tx.tx.gasLimit * stx.tx.tx.gasPrice)
```

Comment:
i know that this is not a refactor, but maybe consider extracting `stx.tx.tx` to val